src/styles.css

```diff
@@ -1 +1,3 @@
 /* You can add global styles to this file, and also import other style files */
+@import "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css";
+@import "../node_modules/components-font-awesome/css/font-awesome.min.css";
```

Comment:
I think you can do `@import "~components-font-awesome/css/font-awesome.min.css";` instead. It tells webpack to resolve it as a module and from the angular-cli webpack config (I'm assuming), it knows to look in node_modules as one of the possible locations.